background.js

```diff
@@ -9,35 +9,11 @@ console.log('imgur heart is loading');
 $(document).ready(function() {
   var button = $(".favorite-image");
 
-  setTimeout(function() {
-    changeUpvote(button);
-    generateTags();
-  }, 500);
-
-  button.click(function() {
-    changeUpvote(button);
-  });
-
-  $(document).click(function() {
-    setTimeout(function() {
-      changeUpvote(button);
-      generateTags();
-    }, 300);
-  });
-
   var points = $(".stats-link.left");
   points.after('<div class="tag-holder"></div>');
-
+  generateTags();
```

Comment:
ok so this needs to happen on click events (and possibly on key press events).  so that it will change without having to refresh.